api/model/src/main/java/org/projectnessie/api/v2/doc/ApiDoc.java

```diff
@@ -48,6 +48,12 @@ public interface ApiDoc {
           + "- \\- (literal minus character) - Identifies the HEAD of the default branch \n"
           + "- name - Identifies the HEAD commit on the named branch\n"
           + "- name@hash - Identifies the 'hash' commit on the named branch.\n"
+          + "- -~3 - The 3rd predecessor commit from the HEAD of the default branch.\n"
+          + "- name~3 - The 3rd predecessor commit from the HEAD of a branch or tag.\n"
+          + "- name@hash^2 - The merge parent of the 'hash' commit of a branch or tag.\n"
+          + "- -*2021-04-07T14:42:25.534748Z - The predecessor commit from the HEAD of the default branch for the given ISO-8601 timestamp.\n"
+          + "- name*2021-04-07T14:42:25.534748Z - The predecessor commit from the HEAD of a branch or tag valid for the given ISO-8601 timestamp.\n"
```

Comment:
I believe we have to specify timestamp-based lookup with more strictness for the sake of clarity. Even commit timestamps are not guaranteed to be monotonous, I think.

Shall we say `The predecessor commit closest to the HEAD of a branch or tag that has its commit time less than or equal to the given ISO-8601 timestamp`?